ldk/javascript/src/types/oliveHelps/whisper.d.ts

```diff
@@ -79,7 +79,7 @@ declare namespace WhisperService {
 
   interface Whisper {
     id: string;
-    close: Common.Readable<undefined>;
+    close: (cb: (err: Error | undefined) => void) => void;
```

Comment:
Why did this get changed?

Reply:
Also wondering this? Is this possibly leftover from when you were testing different ways to type the stringify function?